### ISSUE TITLE
Resolve Regression Timeout

### ISSUE THREAD

**Issue by praveenkuttappan** [Azure/azure-sdk-for-python#11069]
Regression test for azure core needs around 70 minutes now. 35 minutes for latest and 35 minutes for oldest dependency test. Currently test is failing after running for 60 minutes due to default time out error. 
Time out should be set to 120 minutes for now. Regression test is only run during scheduled runs.

**Comment by kurtzeborn** [comment]
@praveenkuttappan Are there any ways we can cut down the time of those tests before we increase the timeout?

**Comment by praveenkuttappan** [comment]
@kurtzeborn Time out change is a temporary fix while I investigate any option to run both latest and minimum dependency tests in parallel. I will have another issue filed to track a permanent fix. Regression test time for core increases over the period based on the test cases added in dependent packages as well as any new dependency since this test run test cases of all dependent packages that takes core as required package.

**Comment by praveenkuttappan** [comment]
Looks like it's already resolved with PR 11105. Closing this old issue.
